packages/x-components/src/x-modules/facets-next/entities/__tests__/editable-number-range-filter-entity.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { createLocalVue } from '@vue/test-utils';
+import Vuex, { Store } from 'vuex';
+import { EditableNumberRangeFilter } from '@empathyco/x-types';
+import { createNextEditableNumberRangeFilter } from '../../../../__stubs__/filters-stubs.factory';
+import { RootXStoreState } from '../../../../store/store.types';
+import { facetsNextXStoreModule } from '../../store/module';
+import { EditableNumberRangeFilterEntity } from '../editable-number-range-filter.entity';
+
+describe('testing EditableNumberRangeFilterEntity', () => {
+  function prepareFacetsStore(): Store<RootXStoreState> {
+    const vue = createLocalVue();
+    vue.use(Vuex);
+    return new Store({
+      modules: {
+        x: {
+          modules: {
+            facetsNext: { ...facetsNextXStoreModule, namespaced: true }
+          },
+          namespaced: true
+        }
+      }
+    });
+  }
+
+  it('allows selecting a filter when filter min is not null', () => {
+    const store = prepareFacetsStore();
+    const filter = createNextEditableNumberRangeFilter('price', { min: 3, max: null });
```

Comment:
With this utility that I updated from the old `createEditableNumberRangeFilter`, the selected value is set immediately depending on the range value. Maybe we can add a third optional parameter to it, `selected = range.min !== null || range.max !== null`, and force `selected` to `false` initially in this test?

Reply:
I've also added checks for the initial filter.selected value which could be redundant but make the tests more explicit. Tell me if we don't need them at all.